README.md

```diff
@@ -1,2 +1,50 @@
-# opg-sirius-supervision-deputy-hub
-OPG sirius supervision deputy hub: Managed by opg-org-infra &amp; Terraform
+# OPG SIRIUS SUPERVISION DEPUTY HUB
+
+### Major dependencies
+
+- [Go](https://golang.org/) (>= 1.16)
+- [docker-compose](https://docs.docker.com/compose/install/) (>= 1.27.4)
+
+#### Installing dependencies locally: 
+- `yarn install`
+- `go mod download`
+-------------------------------------------------------------------
+
+## Local development
+
+The application ran through Docker can be accessed on `localhost:8888/supervision/deputies/public-authority/`.
+
+**Note: Sirius is required to be running in order to authenticate. Ensure that after logging in, you redirect back to the correct port (`8888`)** 
+
+To enable debugging and hot-reloading of Go files:
+
+`docker-compose -f docker/docker-compose.yml -f docker/docker-compose.dev.yml up --build`
```

Comment:
this can just be `docker-compose -f docker/docker-compose.dev.yml up --build` now